modules/apps/sharing/sharing-service/src/main/java/com/liferay/sharing/service/impl/SharingEntryLocalServiceImpl.java

```diff
@@ -200,6 +200,11 @@ public SharingEntry addSharingEntry(
 		return newSharingEntry;
 	}
 
+	@Override
+	public void deleteCompanySharingEntries(long companyId, long classNameId) {
```

Comment:
As you have two parameters, I think the pattern would be using the method name `deleteSharingEntries(`

Reply:
I'm not sure about this name either, but I couldn't use it because that method signature already exists in this class : public void deleteSharingEntries(long classNameId, long classPK).So, I followed this [pattern](https://github.com/liferay/liferay-portal/blob/57ce4750f9161211bbf7641820c71f65e5995bd0/modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/service/impl/FriendlyURLEntryLocalServiceImpl.java#L178).